core/integration/module_cli_test.go

```diff
@@ -562,6 +562,27 @@ func (CLISuite) TestDaggerDevelop(ctx context.Context, t *testctx.T) {
 		require.NoError(t, err)
 		require.Equal(t, "hi from work hi from dep", strings.TrimSpace(out))
 	})
+
+	t.Run("recursive", func(ctx context.Context, t *testctx.T) {
+		c := connect(ctx, t)
+
+		base := c.Container().From(golangImage).
+			WithMountedFile(testCLIBinPath, daggerCliFile(t, c)).
+			WithWorkdir("/work/dep").
+			With(daggerExec("init", "--source=.", "--name=dep", "--sdk=go")).
+			WithExec([]string{"rm", "dagger.gen.go"}).
+			WithWorkdir("/work").
+			With(daggerExec("init", "--source=.", "--sdk=go")).
+			With(daggerExec("install", "./dep")).
```

Comment:
probably doesn't need a test, but do you know if there's any way to mess things up here with symlinks? my intuition is that they should just transparently work, but i don't have anything to back that up with.